.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I'd suggest to add

```
"globals": {
  "H5P": "readonly",
  "H5PEditor": "readonly",
  "H5PIntegration": "readonly"
}
```

to let eslint know that those do not need to be imported, but should only be read and not written to.

Reply:
Perhaps these should be part of NDLA's H5P eslint config? For what it's worth, I don't expect them to be of much use in this repo

Reply:
That's an option, too, yes.

Reply:
https://github.com/NDLANO/eslint-config-ndla-h5p/pull/3

Reply:
Not in a position to say yes or no, but the changes look ok from a technical standpoint!